src/ArrayUtils.js

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2018 New Vector
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * creates a new array with only the unique values of the given array
+ * @param {array} arr the array to deduplicate
+ * @return {array} the deduplicated array
+ */
+export function unique(arr) {
+    const cpy = [];
+    arr.forEach((el) => {
+        if (!cpy.includes(el)) {
+            cpy.push(el);
+        }
+    });
+    return cpy;
+}
```

Comment:
Lodash has one of these: https://lodash.com/docs/4.17.10#uniq which we already have a dependency on: I'd suggest using that (also this is a bit O(n^2) because includes on an array will be linear - a Set ought to be faster).

Reply:
:+1: